pkgs/servers/mail/smtp-to-telegram/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoPackage, fetchFromGitHub, lib }:
+
+buildGoPackage rec {
```

Comment:
Does buildGoModule work here?

Reply:
Neat - I wasn't aware of `buildGoModule`; changed.